src/main/java/org/vaadin/artur/axainputtext/AXAInputText.java

```diff
@@ -1,21 +1,21 @@
-package org.vaadin.artur.paperslider;
+package org.vaadin.artur.axainputtext;
 
 import com.vaadin.flow.component.Component;
 import com.vaadin.flow.component.Tag;
 import com.vaadin.flow.component.dependency.JsModule;
 import com.vaadin.flow.component.dependency.NpmPackage;
 
-@Tag("paper-slider")
-@JsModule("@polymer/paper-slider/paper-slider.js")
-@NpmPackage(value = "@polymer/paper-slider", version = "^3.0.1")
+@Tag("axa-input-text")
+@JsModule("@axa-ch/input-text/lib/index.js")
+@NpmPackage(value = "@axa-ch/input-text", version = "4.3.11")
 /*
  If you wish to include your own JS modules in the add-on jar, add the module
  files to './src/main/resources/META-INF/resources/frontend' and insert an
  annotation @JsModule("./my-module.js") here.
 */
-public class PaperSlider extends Component {
+public class AXAInputText extends Component {
```

Comment:
Should this be `AxaInputText` instead ? I can't find the source, but I do recall that all caps abbreviations are not that convenient for instance to autocomplete, AXAIT is more complex than AIT.

Reply:
Done.